enterprise/server/scheduling/task_leaser/task_leaser.go

```diff
@@ -183,11 +183,10 @@ func (t *TaskLeaser) Close(ctx context.Context, taskErr error, retry bool) {
 	if taskErr == nil || !retry {
 		req.Finalize = true
 	} else {
+		// taskErr != nil && retry
```

Comment:
leftover comment

Reply:
I intentionally left it there as the inverted logic was not obvious to me at first.

Can remove.